### DISCUSSION TITLE
Page not found in Docusaurus template website

### DISCUSSION THREAD

**Discussion by Abhishek-Beta** [rossjrw/pr-preview-action#93] [Q&A] [answered]
I am currently trying to integrate PR previews into my Docusaurus website, which is hosted on GitHub Pages. Although the main site builds and deploys correctly, the PR preview link results in a "Page Not Found" error when clicked. Below are the specifics of my setup and the challenges I'm facing.

**Comment by rossjrw** [comment, marked as answer]
@Abhishek-Beta you haven't provided any specifics so I can't investigate in any detail, but perhaps #14 might be relevant as it was reported by another user of Docusaurus?
